robolectric-resources/build.gradle

```diff
@@ -0,0 +1,17 @@
+dependencies {
+    // Project dependencies
+    compile project(":robolectric-utils")
+    compile project(":robolectric-annotations")
+
+    // Compile dependencies
+    compile "com.ximpleware:vtd-xml:2.11"
+    compile "com.google.guava:guava:19.0-rc2"
+    provided "org.robolectric:android-all:6.0.0_r1-robolectric-0"
```

Comment:
Will be the android.jar from the sdk but since this is a like for like change we can do in a follow up.